README.md

```diff
@@ -56,7 +56,7 @@
 <br>
 
 
-<h2><img height="30" src="/img/AGE.png" style="margin: 0 0 -3px 0">&nbsp;&nbsp;What is Apache AGE?</h2>
```

Comment:
Should this be removed? It is for alignment of heading and icon.

Reply:
That is up to @jbiz805 or @eyab to say. It doesn't seem to have much of an impact and can be addressed later, if needed.

Reply:
Actually, I'm going to leave this until after the merge into PG12. I don't want anyone mistakenly updating the master while we're working on merging.